sql_metadb/derived_tables/po_instance.sql

```diff
@@ -0,0 +1,148 @@
+
+CREATE INDEX ON po_instance (created_date);
+
+CREATE INDEX ON po_instance (bill_to);
+
+CREATE INDEX ON po_instance (ship_to);
+
+CREATE INDEX ON po_instance (pol_instance_id);
+
+CREATE INDEX ON po_instance (pol_instance_hrid);
+
+CREATE INDEX ON po_instance (pol_location_id);
+
+CREATE INDEX ON po_instance (pol_location_name);
+
+CREATE INDEX ON po_instance (pol_location_source);
+
+CREATE INDEX ON po_instance (title);
+
+CREATE INDEX ON po_instance (publication_date);
+
+CREATE INDEX ON po_instance (publisher);
+
+CREATE INDEX ON po_instance (requester);
+
+CREATE INDEX ON po_instance (rush);
+
+CREATE INDEX ON po_instance (selector);
+
+
+--VACUUM ANALYZE  po_instance;
+
+COMMENT ON COLUMN po_instance.manual_po IS 'If true, order cannot be sent automatically, e.g. via EDI';
+
+COMMENT ON COLUMN po_instance.rush IS 'Whether or not this is a rush order';
+
+COMMENT ON COLUMN po_instance.requester IS 'Who requested this material and needs to be notified on arrival';
+
+COMMENT ON COLUMN po_instance.selector IS 'Who selected this material';
+
+COMMENT ON COLUMN po_instance.po_number IS 'A human readable number assigned to PO';
+
+COMMENT ON COLUMN po_instance.po_number_id IS 'UUID identifying this PO';
+
+COMMENT ON COLUMN po_instance.po_line_number IS 'A human readable number assigned to PO line';
+
+COMMENT ON COLUMN po_instance.po_line_id IS 'UUID identifying this purchase order line';
+
+COMMENT ON COLUMN po_instance.vendor_code IS 'The code of the vendor';
+
+COMMENT ON COLUMN po_instance.created_by_username IS 'Username of the user who created the record (when available)';
+
+COMMENT ON COLUMN po_instance.po_workflow_status IS 'Workflow status of purchase order';
+
+COMMENT ON COLUMN po_instance.status_approved IS 'Wether purchase order is approved or not';
+
+COMMENT ON COLUMN po_instance.created_date IS 'Date when the purchase order was created';
+
+COMMENT ON COLUMN po_instance.bill_to IS 'Name of the bill_to location of the purchase order';
+
+COMMENT ON COLUMN po_instance.ship_to IS 'Name of the ship_to location of the purchase order';
+
+COMMENT ON COLUMN po_instance.pol_instance_id IS 'UUID of the instance record this purchase order line is related to';
+
+COMMENT ON COLUMN po_instance.pol_instance_hrid IS 'A human readable number of the instance record this purchase order line is related to';
+
+COMMENT ON COLUMN po_instance.pol_holding_id IS 'UUID of the holdings this purchase order line is related to';
+
+COMMENT ON COLUMN po_instance.pol_location_id IS 'UUID of the location created for this purcase order line';
+
+COMMENT ON COLUMN po_instance.pol_location_name IS 'Name of the purchase order line location';
+
+COMMENT ON COLUMN po_instance.pol_location_source IS 'Wether location is a holdings location or permanent location of the purchase order line';
+
+COMMENT ON COLUMN po_instance.title IS 'Title of the material';
+
+COMMENT ON COLUMN po_instance.publication_date IS 'Date (year) of the material''s publication';
```

Comment:
Contains an additional " ' "

Reply:
@LauSchl: Just to confirm what @natalyapik is saying, this is actually the correct syntax for including a single quote inside a string in SQL. So, if this is the only concern, I think you can approve?

Reply:
Oh ok. My fault and sorry for the delay. I approved it now.